pylintrc

```diff
@@ -15,7 +15,8 @@ max-attributes=15
 const-naming-style=any
 
 disable=duplicate-code,
-        super-init-not-called
+        super-init-not-called,
+        cell-var-from-loop
```

Comment:
any particular reason to add this rule?

Reply:
Just to silent a pylint warning "cell variable define in loop". The warning is not reasonable.  The code which got this warning is in sdk/pynni/nni/gu_tuner/util.py, line 41